recipes/ncbi-cxx-toolkit-public/26/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.9)
+project(PackageTest)
+
+set(CMAKE_CXX_STANDARD 17)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+# the following is not working in the toolkit v26
+#conan_basic_setup(TARGETS)
```

Comment:
does it fail? maybe it's worth reporting as a conan bug?

Reply:
It is not Conan's problem. it is ours.